src/Features/Core/Portable/AddImport/AbstractAddImportCodeFixProvider.cs

```diff
@@ -60,21 +60,19 @@ public sealed override async Task RegisterCodeFixesAsync(CodeFixContext context)
 
             var solution = document.Project.Solution;
 
-            var searchNuGetPackages = solution.Options.GetOption(SymbolSearchOptions.SuggestForTypesInNuGetPackages, document.Project.Language);
-
             var placement = await AddImportPlacementOptions.FromDocumentAsync(document, cancellationToken).ConfigureAwait(false);
 
             var options = new AddImportOptions(
-                context.Options.SearchReferenceAssemblies,
+                context.Options.SearchOptions,
                 context.Options.HideAdvancedMembers,
                 placement);
 
-            var symbolSearchService = options.SearchReferenceAssemblies || searchNuGetPackages
+            var symbolSearchService = options.SearchOptions.SearchReferenceAssemblies
```

Comment:
This looks like a behavior change that impacts getting the `packageSources` below.

Reply:
Will fix.